account_invoice_rounding_by_currency/README.rst

```diff
@@ -0,0 +1,73 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===================================================
+Unit rounded invoice (Swedish rounding) by currency
```

Comment:
Hi @elicoidal the term is already used here https://github.com/OCA/account-invoicing/tree/8.0/account_invoice_rounding

See https://en.wikipedia.org/wiki/Cash_rounding